### PR TITLE
Add unique flag to items

### PR DESCRIPTION
**Pull request by stelar7**

### CODE REVIEW

File: cdragontoolbox/tftdata.py [hunk 1]
```diff
@@ -180,6 +180,7 @@ def parse_items(self, map22):
                 "name": item.getv(0xC3143D66),
                 "desc": item.getv(0x765F18DA),
                 "icon": item.getv("mIconPath"),
+                "unique": item.getv(0x9596A387, false),
```
Comment (benoitryder) [inline cdragontoolbox/tftdata.py:183]: It's `False`, not `false`. ;)

Comment (stelar7) [inline cdragontoolbox/tftdata.py:183]:
```suggestion
                "unique": item.getv(0x9596A387, False),
```

Comment (stelar7) [reply cdragontoolbox/tftdata.py]: Why does python have to be the odd child xD

File: cdragontoolbox/tftdata.py [hunk 2]
```diff
@@ -300,4 +301,3 @@ def main():
 
 if __name__ == "__main__":
     main()
```
Comment (stelar7) [inline cdragontoolbox/tftdata.py:303]:
```suggestion
    main()
    
```

File: cdragontoolbox/tftdata.py [hunk 3]
```diff
@@ -300,4 +301,4 @@ def main():
...
-
+
```
Comment (stelar7) [inline cdragontoolbox/tftdata.py:304]:
```suggestion

```

File: cdragontoolbox/tftdata.py (hunk 1 above)
Comment (stelar7) [reply cdragontoolbox/tftdata.py]: Fixed with ddebdd5d33196999eba0bb0c6715ecd1596af98c

**Pull request merged**